build-model

```diff
@@ -3,6 +3,6 @@ set -e -v
 
 # regenerate the fabric8 java API including k8s custom resources
 pushd $(go env GOPATH)/src/github.com/fabric8io/kubernetes-model/
-git checkout sk8s
+git checkout sk8s-sidecar
```

Comment:
@ericbottard can you please merge `sk8s-sidecar` into `sk8s` and then submit a PR (separate from this one which I'm about to merge) to update this one line?

Reply:
Sure